extensions/pkg/controller/extension/controller.go

```diff
@@ -51,7 +51,11 @@ type AddArgs struct {
 	// with a present operation annotation typically set during a reconcile (e.g. in the maintenance time) by the Gardenlet
 	IgnoreOperationAnnotation bool
 	// ExtensionClass defines the extension class this extension is responsible for.
+	// Deprecated: Use ExtensionClasses instead.
+	// TODO(timuthy): Remove fallback after Gardener v1.116 was released.
```

Comment:
Why 116? Just one release? Then you can also remove it immediately? :D

Reply:
Also fine for me 🙂